setup.py

```diff
@@ -18,10 +18,10 @@
         'Intended Audience :: Information Technology',
         'Intended Audience :: System Administrators',
         'License :: OSI Approved :: Apache Software License',
-        'Programming Language :: Python :: 3.4',
-        'Programming Language :: Python :: 3.5',
         'Programming Language :: Python :: 3.6',
         'Programming Language :: Python :: 3.7',
+        'Programming Language :: Python :: 3.8',
+        'Programming Language :: Python :: 3.9',
```

Comment:
```bash
curl 'https://pypi.org/pypi?%3Aaction=list_classifiers' 2>/dev/null | grep 'Programming Language :: Python :: 3.'
```
Looks like they added 3.10. But I'm guessing Github Actions doesn't have it yet?

Reply:
I think it has the pre-releases, but I don't think the Google libraries this depends on test against it yet. So I was thinking I would add it later.